README.md

```diff
@@ -73,7 +73,7 @@ docker run \
     --base-path=/data-sydney \
     --chain sydney \
     --bootnodes /dns/sun.sydney.ggxchain.io/tcp/30333/p2p/12D3KooWGmopnFNtQb2bo1irpjPLJUnmt9K4opTSHTMhYYobB8pC \
-    --telemetry-url "ws://18.192.225.224:8001/submit 0"
+    --telemetry-url "wss://telemetry.sydney.ggxchain.io/submit 0"
```

Comment:
hmm :c, but we have testnet.telemetry.sydney.ggxchain.io in chainspec. Should we align them?

Reply:
@akorchyn As for me telemetry.sydney.ggxchain.io looks much cleaner. But I wonder if we can make a DNS redirect from old record to the new one? @serkanyazici @humfecki 

We also had different urls to submit telemetry and view them. And we can't change chainspec anymore.

Reply:
Yeah, chain spec is locked. Redirect fine to me, but we need to keep it :)